tests/end_to_end/test_ip_reputation/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,112 @@
+- name: Test manager configuration
+  hosts: wazuh-manager
+  become: true
+  tasks:
+
+    - name: Install apache
+      become: true
+      shell: yum install httpd -y
+
+    - name: Start apache
+      become: true
+      shell: systemctl start httpd
+
+    - name: Install pyhton
+      become: true
+      shell: yum install python39 -y
+
+    - name: Download Alienvault IP
+      become: true
+      shell: >
+        wget https://raw.githubusercontent.com/firehol/blocklist-ipsets/master/alienvault_reputation.ipset
```

Comment:
Can we use `curl` instead of `wget`?

I believe that not all distributions come with `wget` installed by default, however `curl` is much more frequently used.

Reply:
Done in da04dda